docs/docs/virtual.md

```diff
@@ -12,6 +12,12 @@ While Icechunk works wonderfully with native chunks managed by Zarr, there is lo
 
     Currently, Icechunk support virtual references to data stored in `s3` compatible,`gcs`, `http/https`, and  `local` storage backends. Support for [`azure`](https://github.com/earth-mover/icechunk/issues/602) is on the roadmap.
 
+!!! warning "Security concerns with virtual chunks"
```

Comment:
Is "concerns" too strong a word? Maybe considerations?

Reply:
Changed in https://github.com/earth-mover/icechunk/pull/1277/commits/02a19b0d4d13a029e277321af0c3523a457496f4